src/main/scala/nl.knaw.dans.easy.license/LicenseCreatorService.scala

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (C) 2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.license
+
+import javax.servlet.ServletContext
+
+import nl.knaw.dans.lib.logging.DebugEnhancedLogging
+import org.eclipse.jetty.server.Server
+import org.eclipse.jetty.servlet.ServletContextHandler
+import org.scalatra._
+import org.scalatra.servlet.ScalatraListener
+
+import scala.util.Try
+
+class LicenseCreatorService(serverPort: Int, app: LicenseCreatorApp) extends DebugEnhancedLogging {
+  import logger._
+
+  private val server = new Server(serverPort)
+  private val context = new ServletContextHandler(ServletContextHandler.NO_SESSIONS)
+  context.addEventListener(new ScalatraListener() {
+    override def probeForCycleClass(classLoader: ClassLoader): (String, LifeCycle) = {
+      ("anonymous", new LifeCycle {
```

Comment:
I thought we had a better name for this. `license-lifecycle`? That's what we did in pid-generator anyway.